services/upgrader/upgrader_linux.go

```diff
@@ -202,7 +202,7 @@ func UpgradeToE0() error {
 		return err
 	}
 
-	filename, err := download(getCurrentVersionTgzURL(), 1024*1024*300)
+	filename, err := download(getCurrentVersionTgzURL(), 1024*1024*512)
```

Comment:
Wow interesting, I am not sure why we are setting a limit in the first place. The URL seems hardcoded to `releases.mattermost.com`. Are we concerned that an attacker might take over that URL and inject harmful assets? Wouldn't there be a bigger concern if that happens?

cc @jupenur for thoughts.

Reply:
When someone intercepts the request and feeds the server an infinite file, it may be used as a denial of service attack. Seems unlikely, though.

Reply:
It's HTTPS. Should not be possible.

Reply:
Yeah, it's very, very unlikely.

Reply:
We can remove the limit, good catch @agnivade. I don't know why it's there either.